pkg/scheduler/algorithm/priorities/most_requested.go

```diff
@@ -17,12 +17,13 @@ limitations under the License.
 package priorities
 
 import (
+	v1 "k8s.io/api/core/v1"
 	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
-	schedulernodeinfo "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
 )
 
 var (
-	mostResourcePriority = &ResourceAllocationPriority{"MostResourceAllocation", mostResourceScorer}
+	defaultMostRequestedResources = ResourceToWeightMap{v1.ResourceMemory: 1, v1.ResourceCPU: 1}
```

Comment:
similar to above, remove this default value and just use defaultRequestedRatioResources directly in the line below.